src/main/java/org/example/TicTacToe.java

```diff
@@ -0,0 +1,81 @@
+package org.example;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+
+import java.time.LocalDateTime;
+import java.util.*;
+
+public class TicTacToe {
+    public static void main(String[] args) {
+
+        List<Paair> listPaair = new ArrayList<>();
```

Comment:
duzo sie tu dzieje, dacie rade to chociaz podzielic na funkcje ?

Reply:
to tak naprawde startuje gre wiec to tez nie moze zostac w mainie, w prawdziwym mainie wspolnym dla wszystkich powinno byc tylko jedno wywolanie jakiegos obiektu ktory wystartuje gre